### ISSUE TITLE
White Rabbit Scan report: Avoid manual reformatting of "Field Overview" data

### ISSUE THREAD

**Issue by PhilAppleby** [Health-Informatics-UoN/carrot-mapper#613]
CaRROT-Mapper currently requires manual edits in the Scan report "Field Overview" sheet.  (Fraction empty | N unique values | Fraction unique) need changes before upload. Details in comments

**Comment by spco** [comment]
Agreed. We don't make any use of these fields, and so should probably remove all treatment of them. This would require agreement that we truly won't need these fields now or in future (`Type`, `Max length`, `N rows`, `N rows checked`, `Fraction empty`, `N unique values`, `Fraction unique`), before removing the ingestion code and then either deleting the existing values from the DB, or making them optional.

**Comment by spco** [comment]
All 3 can contain non-number values, such as "<10" which aren't currently handled. It'd be easy enough to handle them safely, but we make no use of them so it's insane we still store them. Once upon a time it was all saved in case we found a use for the information.
